lib/ChildUtils.lua

```diff
@@ -6,8 +6,35 @@ end
 
 local ChildUtils = {}
 
+--[[
+	A signal value indicating that a child should use its parent's key, because
+	it has no key of its own.
+
+	This occurs when you return only one element from a functional component or
```

Comment:
Nit: functional components are becoming just 'function components' to differentiate them from... components that aren't broken.

Reply:
Oops, yeah, I missed that!